pkg/controller/kieapp/constants/constants.go

```diff
@@ -291,6 +291,10 @@ const (
 	SUBCOMPONENT_TYPE_INFRA = "infrastructure"
 
 	DefaultDatagridUsername = "infinispan"
+
+	KIE_ADMIN_CREDENTIALS_SECRET = "kie-admin-credentials"
```

Comment:
I think this constant is not needed
```suggestion
```

Reply:
Is used inside checkAndCreateAdminSecret